Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -531,7 +531,7 @@ class BrowserViewController: UIViewController {
         NotificationCenter.default.addObserver(
             self,
             selector: #selector(displayThemeChanged),
-            name: .DisplayThemeChanged,
+            name: .ThemeDidChange,
```

Comment:
`DisplayThemeChanged` is for our `LegacyThemeManager` that we still use in some part of the code. I noticed the BrowserViewController is not following yet the new theme protocol  `Themeable`, so we should use that instead. For more info about how to use `Themeable`, you can have a look at our doc [here](https://github.com/mozilla-mobile/firefox-ios/wiki/Theming-system). We should make sure we `listenForThemeChange` if we make BVC conform to `Themeable`.

Reply:
Ok. Should i register that BVC to both the notification, or just implement the Themeable protocol and remove the old LegacyThemeManager implementation in the controller ?

Reply:
Perfect then i'll wait for that PR to merge